spec/features/page_feature_spec.rb

```diff
@@ -84,13 +75,6 @@ module Alchemy
           end
         end
 
-        context "requested url is only the urlname" do
-          it "then it should redirect to pages url with nested language." do
```

Comment:
No, this is exactly what we want, if the page language and default locale are not the same. If they are the same, then the redirect is not needed.